runbotics/runbotics-orchestrator-ui/src/src-app/hooks/useOptions.ts

```diff
@@ -52,7 +52,8 @@ const UTILS = [
 
 const VARIABLES = [
     'tempFolder',
-    'userEmail'
+    'userEmail',
+    'errorMessages'
```

Comment:
imo rename it to `caughtErrorMessage` we will store just the most recent error message